README.md

```diff
@@ -237,4 +250,4 @@ IGNORE_ACTIVE_CHAN_IDS = tuple()
 
 ## Acknowledgements
 
-Contributors: CloakerSmoker, GeekDude, sjc, Cap'n Odin
\ No newline at end of file
+Contributors: CloakerSmoker, GeekDude, sjc, Cap'n Odin
```

Comment:
Don't ask me what happened here, the lines are identical but git decided to diff them anyway :shrug: